controllers/index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/*
+* 2007-2019 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author PrestaShop SA <contact@prestashop.com>
+*  @copyright  2007-2019 PrestaShop SA
+*  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+require(dirname(__FILE__).'/../../config/config.inc.php');
+require(dirname(__FILE__).'/../../init.php');
```

Comment:
Use a standard index.php

Here you include config.inc.php who launch a connection to database.

And init.php who instantiate an empty FrontController, this is only for retro compatibility with old modules but since 1.5, developers should use ModuleFrontController or use ModuleAdminController depends of the needs.